intro-section.tex

```diff
@@ -3,152 +3,272 @@ \section{Cylc Introduction}
+in the task job log directory.}
+
+A task job can run shell scripting inlined in the suite.rc file, or an
+external program or script, or any combination of the two.  A suite
+sub-directory called \lstinline{bin} is automatically added to your shell
+\lstinline{$PATH} so that scripts residing in it can be called by name,
+
+% changed original script name 'get-host-details' - users may wonder if host
+% details are somehow communicated back to the suite.
+\begin{lstlisting}[language=suiterc]
+[runtime]
+    [[model]]
+        script = run-model
 \end{lstlisting}
 
-The script can then be referenced in the suite.rc file e.g.:
+Alternatively, you can use a specify a full path,
 
 \begin{lstlisting}[language=suiterc]
-[scheduling]
-    [[get_info]]
-        script = get-host-details
+[runtime]
+    [[model]]
+        script = /path/to/my-scripts/run-model
 \end{lstlisting}
 
-In the task definition it is possible to determine what the task will run, as
-well as where and how the task will run. For example by adding a
-\lstinline=[[[remote]]]= section to the task definition it is possible to
-specify the host that will run the command e.g.:
+or you can configure \lstinline{$PATH} in the task's environment,
+\begin{lstlisting}[language=suiterc]
+[runtime]
+    [[model]]
+        script = run-model
+        [[[environment]]]
+            PATH = /path/to/my-scripts:$PATH
+\end{lstlisting}
+
+\more{The last method makes more sense if there are multiple task scripts in
+    the same location, then you can set \lstinline{$PATH} once in a {\em
+    family} (see below) inherited by all the tasks
+that need those scripts.}
+
+An optional \lstinline=[[[remote]]]= sub-section determines {\em where} a task
+will run (defaults to localhost), 
 
 \begin{lstlisting}[language=suiterc]
-[scheduling]
-    [[get_info]]
-        script = get-host-details
+[runtime]
+    [[model]]
+        script = run-model
         [[[remote]]]
             host = supercomputer
 \end{lstlisting}
 
-It is also possible to pass information to the script, for instance in the
-following example the script ``get-host-details'' is provided with the
-environment variable ``FOO'' set to ``bar''.
+\more{Cylc doesn't install files to remote hosts for you.  If a task host does
+    not share a filesystem with the suite host, make sure you install any files
+    needed by the tasks that run there.  Rose (\ref{Rose}) can help automate
+file installation.}
 
+An optional \lstinline=[[[job]]]= sub-section determines {\em how} to run the
+task (default to a shell "background" subprocess).
 \begin{lstlisting}[language=suiterc]
-[scheduling]
-    [[get_info]]
-        script = get-host-details
+[runtime]
+    [[model]]
+        script = run-model
+        [[[remote]]]
+            host = supercomputer
+        [[[job]]]
+            batch system = pbs
+\end{lstlisting}
+
+\more{Cylc can submit jobs to various batch systems such as PBS and SLURM. With
```

Comment:
I agree with Hilary here - a large part of cylc is abstracting away interacting with batch submission. I think it's worth it, given that we can't actually do an advanced tutorial (that would run) about it.